stories/assets/js/filter_seach_bar.js

```diff
@@ -26,22 +26,22 @@ const toggleFilter = function () {
     }
     if (el.is(':checked')) {
       const optionValue = $(this).siblings().text();
-      const chips = $(currentChipsWrapper).find('.chips:first-child').clone();
+      const chips = $('<a class="chips chips__cross" href="#" role="button">demo</a>').clone();
       chips.text(optionValue);
       chips.attr({'option-name': eleId });
       $(currentChipsWrapper).append(chips);
 
-      if (currentChipsWrapper.find('.chips__cross').length > 1) {
+      if (currentChipsWrapper.find('.chips__cross').length > 0) {
```

Comment:
@Ashish-srij Can we change this to `.hasClass()` to check if class exists?

Reply:
@shishirphadke we can not use hasClass because we are finding the child element.